content/news/Reproc-Update-1-C2-C13-C3.md

```diff
@@ -0,0 +1,79 @@
+</div>
+<img style="text-align: center" src="images/news/reproc-status-1-minor.png" width="100%" align="center"  class="img-responsive">
+</div>
+
+<br>
+
+Notable changes to the data from campaigns 2, 3, and 13:
+
+- In the original C2 processing, lightcurves were not produced, and TPFs were "Type-1", which had poor WCS information and other limitations compared to the "Type-2" TPFs that were produced in later campaigns. (For details on the difference between Type-1 and Type-2 TPFs, see [section 2.4 of the K2 Handbook](https://archive.stsci.edu/k2/manuals/k2_handbook.pdf#page=15).) As a result of reprocessing, Type-2 TPFs, lightcurves (both long- and short-cadence), and cotrending basis vectors (CBVs) are now available for C2.
+
+- One of the major improvements from the global reprocessing is the use of an algorithm called "Dynablack", which is essentially a more sophisticated way to perform the CCD pixel-level calibration that accounts for time-varying, instrument-induced artifacts in the pixel data. One benefit of Dynablack is that is is able to correct crosstalk from the fine guidance sensors, resulting in much better background correction for certain channels. As shown below, for channel 41 in the original C2 processing, the old CCD calibration method resulted in varying background level as a function of position on the CCD. In the reprocessing, the use of Dynablack resulted in a significantly better correction and more uniform background levels.
+
+<div class="thumbnail" style="width: 80%; display: inline-block;">
+<div class="caption">
+<i>The residuals in the background correction fit as a function of row position on the CCD. <b>Left:</b> The original C2 processing. <b>Right:</b>The same figure from the C2 reprocessing. (Note the different y-axis scale between the two figures.)</i>
+</div>
+<img style="text-align: center" src="images/news/c2-dynablack-correction.png" width="100%" align="center"  class="img-responsive">
+</div>
+
+<br>
+
+- As part of the reprocessing, we are now producing short-cadence lightcurves for all objects that were observed in short-cadence mode. We note that no work was able to be done to adapt the detrending method used for Kepler short-cadence lightcurves to those from K2, and thus on a whole they are expected to be of varying quality. However, we are producing them with the expectation that at least some objects will be of high quality, and/or their existence will spurn more detailed investigation. Below we show the short-cadence PDC lightcurve for EPIC 206003187, an RR Lyrae-type variable observed in C3. While there are still residual systematics near thruster-firing events, simply removing those points near thruster firing events results in a well-detrended phased light curve with a clearly visible [Blazhko effect](https://en.wikipedia.org/wiki/Blazhko_effect).
```

Comment:
I don't think it's true that "short cadence detrending was not adapted for K2", because presumably the short cadence detrending does benefit from changes made to PDC to make it work on long cadence K2 data?  I'd rephrase the first sentence to avoid down-playing the effort!